src/client/components/localisation/nugus_robot/head/view_model.ts

```diff
@@ -29,7 +29,7 @@ export class HeadViewModel {
     const { geometry, materials } = this.neckGeometryAndMaterial
     const mesh = new Mesh(geometry, materials)
     mesh.position.set(0, 0.051, 0.08)
-    mesh.rotation.set(0, this.model.motors.headPan.angle, 0)
+    mesh.rotation.set(0, this.model.motors.headPan.angle, this.model.motors.headTilt.angle)
```

Comment:
I very much doubt that you should be doing both of these rotations at once. Our servos don't have two degrees of freedom

Reply:
That's nice to know. If we call the y axis up, which axis can the head rotate on?

Reply:
The axis of rotation will depend on the local frame of reference. The way to think about it is that the robot is made up of rigid bodies. Each of these is a part that is solidly attached to a single servo. This servo can only rotate around a single axis. All of the parts should just be rigid bodies in rigid bodies all centered on the torso. So for the head, there are three rigid bodies at play, the torso, which is always just itself, the neck, which is just a single servo with no printed parts, and the head. So you can rotate the neck, and you can rotate the head, but this is two single rotations.

Reply:
So if you look in the Darwin model, all rotations are around the x or y axis. That’s because the rotation is local to the frame of reference. The rotation of the head may be x or y (or z) depending on the orientation of the actual part itself.